WalletWasabi.Gui/CrashReport/Views/CrashReportWindow.xaml

```diff
@@ -0,0 +1,28 @@
+<cont:WasabiWindow xmlns="https://github.com/avaloniaui"
+                  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                  xmlns:cont="clr-namespace:AvalonStudio.Shell.Controls;assembly=AvalonStudio.Shell"
+                  xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+                  xmlns:wasabi="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
+                  xmlns:behaviors="clr-namespace:WalletWasabi.Gui.Behaviors;assembly=WalletWasabi.Gui"
+                  x:Class="WalletWasabi.Gui.CrashReport.Views.CrashReportWindow"
+                  Icon="avares://WalletWasabi.Gui/Assets/WasabiLogo256.png"
+                  Title="{Binding Title}"
+                  MinWidth="{Binding MinWidth}" MinHeight="{Binding MinHeight}"
+                  Width="{Binding MinWidth, Mode=OneTime}" Height="{Binding MinHeight, Mode=OneTime}"
+                  FontFamily="{DynamicResource UiFont}" FontSize="14"
+                  Foreground="{DynamicResource ThemeForegroundBrush}"
+                  WindowState="{Binding WindowState}"
+                  Classes="ToolWindow"
+                  CanResize="False">
+  <i:Interaction.Behaviors>
+    <behaviors:CloseWindowCommandBehavior Command="{Binding OkCommand}" />
+  </i:Interaction.Behaviors>
+  <DockPanel LastChildFill="True" Margin="25,0,25,25">
+    <StackPanel DockPanel.Dock="Bottom" Orientation="Horizontal" HorizontalAlignment="Right" Spacing="5" Margin="0,25,0,0">
+      <Button MinWidth="135" Height="35" Command="{Binding OpenLogCommand}">Open logs</Button>
+      <Button MinWidth="135" Height="35" Background="{DynamicResource ThemeAccentBrush}" Command="{Binding OkCommand}">Ok</Button>
+    </StackPanel>
+    <TextBox DockPanel.Dock="Top" Text="{Binding Message}" MaxHeight="50" IsReadOnly="True" TextWrapping="Wrap" AcceptsReturn="True" FontWeight="Bold" FontSize="18" Margin="0,0,0,25" BorderThickness="0" />
+    <TextBox Text="{Binding Details}" IsReadOnly="True" Margin="0 8" TextWrapping="Wrap" AcceptsReturn="True" BorderThickness="0" />
```

Comment:
Isn't this better:

```xml
<TextBlock Text="{Binding Details}" Margin="0 8" TextWrapping="Wrap" />
```

wdyt? It's not editable and it looks more like a standard dialog window.

Reply:
Ok. For the details let's not allow selection, it is a constant text anyway.

Reply:
Fixed